### ISSUE TITLE
no image preview running in google colab

### ISSUE THREAD

**Issue by xmuninx** [TinyTerra/ComfyUI_tinyterraNodes#28]
I install using git clone in colab but image preview doesn't work, and run comfyui with "--preview-method auto"

<img width="701" alt="Captura de pantalla 2023-08-12 133558" src="https://github.com/TinyTerra/ComfyUI_tinyterraNodes/assets/142112978/0fa5cdad-6aa7-4aa4-a141-1151a61a4f9f">

**Comment by itsmeherefolks** [comment]
Thanks for this great node package!

I have the same problem. The vanilla preview node is showing the image but the ttn nodes imageOutput and pipeKSampler are showing no output without "--preview-method". With "preview-method auto" the sampler shows a grainy image and the imageOutput still shows no image.

I am using the latest  main commit / ac188ae0ddf765dfca33bba87b4f0cd4c40e8d97.
under
Manjaro Linux with an Anaconda environment ("nvidia/label/cuda-11.8.0" cuda-toolkit + pytorch torchvision torchaudio pytorch-cuda=11.8 -c pytorch -c nvidia)
with
Nvidia3060 12GB

![image](https://github.com/TinyTerra/ComfyUI_tinyterraNodes/assets/33434783/aff406f5-e673-4642-b5a1-e9be5fbfd6fc)

**Comment by TinyTerra** [comment]
This is very strange, since it should be outputting in the same way the default preview does... 
unfortunately I don't have colab set up to be able to troubleshoot.. but i'll try to have more of a look into it

**Comment by itsmeherefolks** [comment]
Thanks! Only for clarification, I am using a local Linux machine. It  might be a Linux related problem. Please tell me if i can support you with further information.

**Comment by zixuzhuang** [comment]
same problem, running on ubuntu 18.04 in local server.

**Comment by bridgesense** [comment]
Yeah, so far, I'm liking this better than the efficiency package, but there's no preview working on Ubuntu 22.04 either.

**Comment by TinyTerra** [comment]
Do the images get saved correctly at least? I'm in the middle of a major update so I'll see if I can finally squash this is issue alongside it. Anymore info you think is relevant would be helpful

**Comment by bridgesense** [comment]
Yes, the preview images are saving to the temp directory as they should. 

There aren't any errors showing up in the terminal, but I'm not well-versed on debugging ComyUI either.

I imagine this is probably an issue with a library version. For example, I had to downgrade my Pillow version in order to get XY plot working.

**Comment by MrYutz** [comment]
I can validate that on a Linux system the images don't show up in the Ksampler.

The images are being created in /stable-diffusion/temp/ttNpreview_0000X.png

I just ran 4 tests and have images 1-4 present, but not displayed in Comfy.
